### DISCUSSION TITLE
Spin Polarized Optimization + Frequencies

### DISCUSSION THREAD

**Discussion by AHalabiD** [tblite/tblite#206] [Q&A]
Dear friends,

I'm currently trying to do spin polarized optimization with a cationic radical system. The calculations are not running properly, and no HOMO-LUMO energies are printed at all.

Is it possible to do this wiht TBLite?

**Comment by haneug** [comment]
Could you please provide an example where the polarized calculation fails?

**Reply by AHalabiD** [reply]
Dear professor Haneug, 

The output of the calculations come out in a "weird" format. Its almost the correct one with some things are missing and some bizarre characters appear (img 1).  Also, the HOMO and LUMO energies are missing completely in the doc. This was running the following command in powershell with the subsequent example of output file with the weird characters and errors: 

.\xtb $inputFile --gfn 2 --spinpol --tblite  --chrg 1 --uhf 1 --ohess --parallel 12 *> "./Outputs/$outputFile" 
Image

![OutputSpinPol](https://github.com/user-attachments/assets/f87ee28e-2acd-4558-8852-ef058887f37e)

Thank you very much for your time!

**Reply by haneug** [reply]
Your powershell seems to have problems with the special characters in the output like Δ or ⁻¹.
You can try to enable UTF-8. I am myself not familiar with powershell but you might want to try:
`$OutputEncoding = [System.Text.Encoding]::UTF8`
and see if it helps with this problem.

Regarding the HOMO-LUMO gap: This is due to the incomplete integration of the tblite into xtb. You can access this data by performing a sinlgepoint calculation on the optimized structure with the tblite standalone and spin-polarization.
